audit/src/test/java/com/networknt/audit/AuditHandlerTest.java

```diff
@@ -358,6 +359,38 @@ public void testAuditWith401ServiceId() throws Exception {
         verifyAuditInfo("serviceId", "com.networknt.petstore-1.0.0");
     }
 
+    @Test
+    public void testAuditWith200TimestampFormatted() throws Exception {
```

Comment:
Is it also possible to test the situation when adds the invalid format, with current solution, I would expect the app failing to start.

Reply:
> Is it also possible to test the situation when adds the invalid format, with current solution, I would expect the app failing to start.

I will add a test for that, but the server won't stop because of the format error.